bip-0158.mediawiki

```diff
@@ -344,7 +344,7 @@ Light client: [https://github.com/lightninglabs/neutrino]
 
 Full-node indexing: https://github.com/Roasbeef/btcd/tree/segwit-cbf
 
-Golomb-Rice Coded sets: https://github.com/btcsuite/btcutil/blob/master/gcs
+Golomb-Rice Coded sets: https://github.com/btcsuite/btcd/tree/master/btcutil/gcs
```

Comment:
LGTM. Per https://github.com/btcsuite/btcutil/tree/master, "This package is now a sub-package in the [btcutil](https://github.com/btcsuite/btcd/tree/master/btcutil) directory of the [btcd](https://github.com/btcsuite/btcd) repository."